### PR TITLE
51 add an admin controller

### PR DESCRIPTION
**Pull request by cadsilva**

### CODE REVIEW

File: src/main/java/SYSC6/Project/Main_Controller.java [hunk 1]
```diff
@@ -83,6 +92,35 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
...
+        List<User> user = new ArrayList<User>();
+
+        userRepository.findAll().forEach(user::add);
+
+        if(user.isEmpty()){
+            return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+        }
+        return new ResponseEntity<>(user,HttpStatus.OK);
+
+    }
```
Comment (aimansharif) [inline src/main/java/SYSC6/Project/Main_Controller.java:145]: remove whitespace.

File: src/main/resources/templates/admin_portal.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+</div>
+<form action="#" th:action="@{/LogOut}" method="post">
+    <button type="submit" class="float">Log out</button>
+</form>
+<form action="#" th:action="@{/view_users}" method="post">
+    <button type="submit" class="float">View list of Users</button>
+</form>
+
+
+</body>
```
Comment (aimansharif) [inline src/main/resources/templates/admin_portal.html:23]: remove unnecessary whitespace

**Review by aimansharif — APPROVED** [review]
Other than the unnecessary whitespace, looks good!

**Review by NicolasDuciaume — CHANGES_REQUESTED** [review]
Remove userRepository calls from within Main_controller, Rest looks good

**Review by NicolasDuciaume — APPROVED** [review]
Thank you
